test/spec/modules/vidoomyBidAdapter_spec.js

```diff
@@ -0,0 +1,209 @@
+            'context': 'outstream',
+            'playerSize': [400, 225],
+          }
+        },
+        'adUnitCode': 'code2',
+      }
+    ];
+
+    let bidderRequest = {
+      refererInfo: {
+        numIframes: 0,
+        reachedTop: true,
+        referer: 'http://example.com',
+        stack: ['http://example.com']
+      }
+    };
+
+    const request = spec.buildRequests(bidRequests, bidderRequest);
+
+    it('sends bid request to our endpoint via GET', function () {
+      expect(request[0].method).to.equal('GET');
+      expect(request[1].method).to.equal('GET');
+    });
+
+    it('attaches source and version to endpoint URL as query params', function () {
+      expect(request[0].url).to.include(ENDPOINT);
+      expect(request[1].url).to.include(ENDPOINT);
+    });
+
+    it('only accepts first width and height sizes', function () {
+      expect(request[0].url).to.include('w=300');
+      expect(request[0].url).to.include('h=250');
+      expect(request[0].url).to.not.include('w=200');
+      expect(request[0].url).to.not.include('h=100');
+      expect(request[1].url).to.include('w=400');
+      expect(request[1].url).to.include('h=225');
+    });
+
+    it('should send id and pid parameters', function () {
+      expect(request[0].url).to.include('id=123123');
+      expect(request[0].url).to.include('pid=123123');
+      expect(request[1].url).to.include('id=456456');
+      expect(request[1].url).to.include('pid=456456');
+    });
+  });
+
+  describe('interpretResponse', function () {
+    const serverResponseVideo = {
+      body: {
+        'vastUrl': 'https:\/\/vpaid.vidoomy.com\/demo-ad\/tag.xml',
+        'mediaType': 'video',
+        'requestId': '123123',
+        'cpm': 3.265,
+        'currency': 'USD',
+        'width': 0,
+        'height': 300,
+        'creativeId': '123123',
+        'dealId': '23cb20aa264b72',
+        'netRevenue': true,
+        'ttl': 60,
+        'meta': {
+          'mediaType': 'video',
+          'rendererUrl': 'https:\/\/vpaid.vidoomy.com\/outstreamplayer\/bundle.js',
+          'advertiserDomains': ['vidoomy.com'],
+          'advertiserId': 123,
+          'advertiserName': 'Vidoomy',
+          'agencyId': null,
+          'agencyName': null,
+          'brandId': null,
+          'brandName': null,
+          'dchain': null,
+          'networkId': null,
+          'networkName': null,
+          'primaryCatId': 'IAB3-1',
+          'secondaryCatIds': null
+        }
+      }
+    }
+
+    const serverResponseBanner = {
+      body: {
+        'ad': '<iframe src=\'https:\/\/vidoomy.com\/render\/ad.html\' width=\'300\' height=\'250\' frameborder=\'0\' scrolling=\'no\' marginheight=\'0\' marginwidth=\'0\' topmargin=\'0\' leftmargin=\'0\'><\/iframe>',
+        'mediaType': 'banner',
+        'requestId': '123123',
+        'cpm': 4.94,
+        'currency': 'USD',
+        'width': 250,
+        'height': 300,
+        'creativeId': '123123',
+        'dealId': '230aa095cea76b',
```

Comment:
Does your bidder plan on supporting deals? I don't see the property included in `interpretResponse`.